src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -14,7 +14,8 @@ public default void enable() {}
    */
   public default void runVoltage(double volts) {}
 
-  public default void runVelocity(double setPoint) {}
+  /** Run closed loop at the specified velocity */
+  public default void runVelocity(double velocityRPM) {}
```

Comment:
RPM = Revolutions Per Minute?

Reply:
Yes.  I was looking at advantagekit example to get idea of how this all works.  I see that we use RPM for the "public" interface for shooter subsystem.  Then converter to rad/sec inside code, then back to RPM again to call the sparkmax pid api.  I think RPM unit is easier to reason about(like mile vs km)? And RPM  matches hw api calls.  The wpilib math stuff uses rad/sec.